projects/packages/my-jetpack/_inc/components/product-detail-card/button.jsx

```diff
@@ -5,35 +5,16 @@ import React from 'react';
 import { Button } from '@wordpress/components';
 import { Spinner } from '@automattic/jetpack-components';
 
-const ProductDetailButton = ( {
-	children,
-	className,
-	href,
-	isLink,
-	isLoading,
-	onClick,
-	isPrimary,
-	isSecondary,
-} ) => {
+const ProductDetailButton = ( { children, className, href, isLoading, onClick } ) => {
 	return (
-		<Button
-			onClick={ onClick }
-			isLink={ isLink }
-			isPrimary={ isPrimary }
-			isSecondary={ isSecondary }
-			className={ className }
-			href={ href }
-		>
+		<Button isPressed onClick={ onClick } className={ className } href={ href }>
```

Comment:
wondering whether `isPressed` does make sense here. I think we should tweak the styles using CSS instead.

Reply:
Why? We use `isPressed` around the entire code, to apply black layout.

Reply:
> We use isPressed around the entire code,...

It doesn't mean it's right :-)

Maybe I'm wrong, but `isPressed` means that, for some reason, the button is pressed. It doesn't seem the case.
It's like we apply `disable` or `::hover` simply because we'd like to apply at the button the same styles that these states do. The button is not disabled, hovered, or pressed, but simply it requires different styles.